policy/approval/approve.go

```diff
@@ -304,6 +304,10 @@ func (r *Rule) isApprovedByConditions(ctx context.Context, prctx pull.Context) (
 // FilteredCandidates returns the potential approval candidates and any
 // candidates that should be dimissed due to rule options.
 func (r *Rule) FilteredCandidates(ctx context.Context, prctx pull.Context) ([]*common.Candidate, []*common.Dismissal, error) {
+	if r.Requires.Count == 0 {
```

Comment:
nit: for consistency with the check in `IsApproved`, I'd suggest using `<= 0` here

Reply:
Sure! 0b558a3b9d81c0b941b7faaec147e772ab552423